integr8ly/integreatly-operator
# [MGDAPI-3029] test: automate B01B - verify product logins

# Issue link

https://issues.redhat.com/browse/MGDAPI-3029
# What

Automating B01B test case for checking product logins for RHOAM for dedicated admin and developer users

# Verification steps



* Install RHOAM
```
export INSTALLATION_TYPE=managed-api
make cluster/deploy
```
* Once instllation completes, run the idp script
```
 PASSWORD=Password1 DEDICATED_ADMIN_PASSWORD=Password1 ./scripts/setup-sso-idp.sh
```
* Run the test
```
INSTALLATION_TYPE=managed-api BYPASS_STORAGE_TYPE_CHECK=true TEST="B01B" make test/e2e/single 
```
* Verify test passes
